sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_client.py

```diff
@@ -40,13 +41,13 @@ class ConversationAnalysisClient(
      https://:code:`<resource-name>`.cognitiveservices.azure.com). Required.
     :type endpoint: str
     :param credential: Credential needed for the client to connect to Azure. Required.
-    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :type credential: ~azure.core.credentials.TokenCredential
```

Comment:
Does your `AzureKeyCredential` extend `TokenCredential`? In .NET, ours doesn't.

Reply:
No, but we're not able to generate with clients that accept both `AzureKeyCredential` and `TokenCredential`. So we generate with `TokenCredential` support only, but you'll see in the _patch.py file where I fix this to accept either: https://github.com/Azure/azure-sdk-for-python/blob/3a067ece82aafaa628491702f92341c253e7fbdb/sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/authoring/aio/_patch.py#L54